### DISCUSSION TITLE
Is accessibility tag support possible for screen readers on objects in Winutil? Also Full black theme.

### DISCUSSION THREAD

**Discussion by wise1453** [ChrisTitusTech/winutil#2327] [Ideas]
I am a visually impaired person who knows the computer at the Power User level. My vision level is low. I have little vision. In lens mode, I use the full-frame width Windows magnifier and NVDA screen reader with under-mouse reading mode active.

Your application is very nice. However, I have a hard time using it because I cannot read objects under the mouse. Even though most object descriptions are in the form of pop-up hints and these hints are spoken by my screen reader, it's not enough. Because I can use it very slowly. There are also many objects without clues. I can't get these to read at all. We cannot provide full accessibility with the keyboard. 

My request from you is that you create accessibility labels for objects during the coding phase. This post-processing can be used with our screen readers. 

Visually impaired people may seem less. But, contrary to what you know, they are very good at using computers. Computer is life for us. 

Finally.

Can you please arrange the text and objects to have a black background and white or yellow color tones? You can offer options. Like theme support.

**Comment by Marterich** [comment]
Hi @wise1453 

Thank you for this post. 

Regarding the accessibility labels for the use of screen readers I'll add @og-mrk to this because he is at the moment working on a refactor of the UI to make it more manageable. Maybe he can try to implement this.

Regarding the theming options. 
At the moment when winutil starts it automatically selects a theme depending on your system preferences (dark/light) and the performance of your GPU. At the moment it's not really possible to easily change the theme yourself without a bit of scripting knowledge. 

In the future I hope we can manage to get something like a theme selector in the application to work or at the very least, provide a script argument to override the automated way of determining the color scheme.

**Reply by og-mrk** [reply]
Linking this discussion to #2226 Progress Tracking Issue.

Thanks for bringing light on this suggestion for me @Marterich 😄
Will try my best to figure out the implementation details 👍
